envs/monkey_zoo/blackbox/log_handlers/logs_downloader.py

```diff
@@ -0,0 +1,22 @@
+from envs.monkey_zoo.blackbox.log_handlers.monkey_log import MonkeyLog
+
+
+class LogsDownloader(object):
+
+    def __init__(self, island_client, log_dir_path):
+        self.island_client = island_client
+        self.log_dir_path = log_dir_path
+        self.monkey_log_paths = []
+
+    def download_monkey_logs(self):
+        print("Downloading each monkey log.")
+        all_monkeys = self.island_client.find_monkeys_in_db(None)
```

Comment:
Making `None` a default argument value is better than passing `None`; see Clean Code chapter 7 section "Don’t Pass Null".

Reply:
Neither passing `None` nor `None` value by default solves the problem described in clean code. Correct solution would be to forbid passing `None` **at all**.